.github/workflows/build.yaml

```diff
@@ -51,3 +52,21 @@ jobs:
       - name: Run npm tests
         run: |
           npm test
+
+      - name: Archive app artifacts
+        uses: actions/upload-artifact@v4
+        with:
+          name: app
+          # These are all of the paths referenced in spin.toml
```

Comment:
I'm a bit concerned about the maintenance implications here but it's not a blocker

Reply:
Yes, I am as well... I took inspiration from another website/app (Finicky Whiskers) as I liked the flow of "build, save artifacts, deploy artifacts" but this site has many more files to track.  Maybe all we need is a better pattern matching approach... eg something as simple as a "not node_modules" negation.

Reply:
https://github.com/spinframework/spin-docs/issues/44